force-app/main/default/lwc/edaSettingsNavigation/edaSettingsNavigation.js

```diff
@@ -78,7 +81,7 @@ export default class EdaSettingsNavigation extends LightningElement {
             storeErrors: stgStoreErrorsTitle,
         },
         programSettings: {
-            programSettings: "Programs",
+            programSettings: stgProgramsSettingsNav,
             programAutoEnrollmentMappings: "Program Auto-Enrollment Mappings",
```

Comment:
Since we're adding program auto enrollment mappings, we should also be setting this label.

Reply:
Not sure if I am following. I thought we are adding auto enrollment to the same page as other program settings?

Reply:
I mean on line 85, you have a label for the table that is the title of the display. That is "Program Auto-Enrollment Mappings". So we should include that in the nav.